unittest: pass Non IDR frame to decode after flush

LindaYu17 opened the pull request:
this is used to verify yami's robustness

Signed-off-by: Linda Yu <linda.yu@intel.com>

LindaYu17 committed:
unittest: pass Non IDR frame to decode after flush

this is used to verify yami's robustness

Signed-off-by: Linda Yu <linda.yu@intel.com>

xuguangxin commented on decoder/DecoderApi_unittest.cpp:
```diff
@@ -316,6 +316,70 @@ TEST_P(DecodeApiTest, Flush)
+    int32_t inFrames = 0;
+    int32_t outFrames = 0;
+
+    for (int i = 0; i < size; i++) {
```
This will flush every frame, it does not simulate the real case. It's better to change to this: Suppose we have N frame, we flush the decoder at frame 1 and play to end, seek to begin and flush at frame 2 ..etc

LindaYu17 commented:
@xuguangxin updated！

LindaYu17 committed:
unittest: add mpeg2 test frames

Signed-off-by: Linda Yu <linda.yu@intel.com>

LindaYu17 committed:
vaapidecoder_h265: return invalid if 1st frame is not idr

if the frist frame of a new stream is not idr, return invalid data
directly but not process it.

Signed-off-by: Linda Yu <linda.yu@intel.com>
